### PR TITLE
TIP-652: import export rework

### PR DESCRIPTION
**Pull request by juliensnz**
Changes on the doc following this PR: https://github.com/akeneo/pim-community-dev/pull/5366

### CODE REVIEW

File: cookbook/import_export/create-connector.rst [hunk 1]
```diff
@@ -144,35 +144,28 @@ Configure the UI for our JobParameters
 
 At this point, the job is usable in command line though it cannot be configured via the UI.
 
-We need to write a service providing the form type configuration for each parameter of our JobParameters instance.
+We need to provide a form name to the frontned to be able to render it properly. If your connector doesn't require extra fields you use the basic froms already in the Akeneo:
```
Comment (phaseinducer) [inline cookbook/import_export/create-connector.rst]: "in the Akeneo"?

Comment (damien-carcel) [inline cookbook/import_export/create-connector.rst]: frontned => frontend

File: cookbook/import_export/create-connector.rst [hunk 2]
```diff
@@ -144,35 +144,28 @@ Configure the UI for our JobParameters
...
-
-.. literalinclude:: ../../src/Acme/Bundle/DummyConnectorBundle/Job/JobParameters/FormConfigurationProvider/DummyExport.php
-    :language: php
+.. literalinclude:: ../../src/Acme/Bundle/DummyConnectorBundle/Resources/config/providers.yml
+    :language: yaml
     :linenos:
+    :lines: 1-6
 
-This service is configured using a tag and it implements ``supports()`` method to be used for our job only.
+If you want to go further, you can create your own form and re-use common parts provided by default. Here are an example of a full form configuration:
```
Comment (damien-carcel) [inline cookbook/import_export/create-connector.rst]: Here **is** an example

File: cookbook/import_export/create-connector.rst [hunk 3]
```diff
@@ -144,35 +144,28 @@ Configure the UI for our JobParameters
...
 
-.. literalinclude:: ../../src/Acme/Bundle/DummyConnectorBundle/Resources/config/job_parameters.yml
+.. literalinclude:: ../../src/Acme/Bundle/DummyConnectorBundle/Resources/config/form_extensions.yml
     :language: yaml
     :linenos:
-    :lines: 1,4-6,25-33
 
-Translate Job and Step labels in the UI
----------------------------------------
+Register it in the form provider and you are good to go !
```
Comment (damien-carcel) [inline cookbook/import_export/create-connector.rst]: no space before `!`

**Pull request merged**